extensions/amp-form/0.1/amp-form.js

```diff
@@ -436,29 +447,75 @@ export class AmpForm {
    */
   handleXhrSubmit_(varSubsFields, trust) {
     this.setState_(FormState_.SUBMITTING);
-
-    const p = this.doVarSubs_(varSubsFields)
-        .then(() => {
-          this.triggerFormSubmitInAnalytics_('amp-form-submit');
-          this.actions_.trigger(
-              this.form_, 'submit', /* event */ null, trust);
-          // After variable substitution
-          const values = this.getFormAsObject_();
-          // At the form submitting state, we want to display any template
-          // messages with the submitting attribute.
-          this.renderTemplate_(values);
-        })
-        .then(() => this.doActionXhr_())
-        .then(response => this.handleXhrSubmitSuccess_(response),
-            error => {
-              return this.handleXhrSubmitFailure_(/** @type {!Error} */(error));
-            });
-
+    const varSubPromise = this.doVarSubs_(varSubsFields);
+    let p;
+    if (this.ssrTemplateHelper_.isSupported()) {
+      p = varSubPromise.then(() => {
+        this.actions_.trigger(
+            this.form_, 'submit', /* event */ null, trust);
+      }).then(() => {
+        return this.ssrTemplateHelper_.fetchAndRenderTemplate(
+            this.form_,
+            this.handleSsrTemplateSuccess_.bind(this),
+            this.handleSsrTemplateFailure_.bind(this)
+        );
+      });
+    } else {
+      p = varSubPromise
+          .then(() => {
+            this.submittingWithTrust_(trust);
```

Comment:
This does not match the old behavior.

Reply:
I'm a little confused about some of these changes. Why doesn't this trigger form analytics if the viewer doesn't support rendering templates? These form features seem unrelated to whether the viewer or the page itself render the form's templates.

It also looks like a merge conflict happened and a recent feature was removed, with the line `this.triggerFormSubmitInAnalytics_('amp-form-submit');`

Reply:
analytics will not be supported in AMP4Email. I'll share with you the docs that capture this. Sorry for the lack of context.